include/field_player_avatar.h

```diff
@@ -6,14 +6,25 @@
 #include "field_types_def.h"
 #include "save.h"
 
+typedef enum PlayerAvatarFlags {
+    AVATAR_FLAG_UNK0 = (1 << 0),
+    AVATAR_FLAG_UNK1 = (1 << 1),
+    AVATAR_FLAG_UNK2 = (1 << 2),
+    AVATAR_FLAG_DISMOUNTING_BIKE_BLOCKED = (1 << 3), // Blocks the player from getting off their bike (technically, you can use this to prevent them from getting on it, too)
```

Comment:
wouldnt this be better named as FORCE_BIKE_STATE?

Reply:
i'll do `LOCK_BIKE_STATE`